### ISSUE TITLE
Is the project maintained?

### ISSUE THREAD

**Issue by gerardbosch** [KuenzelIT/ulauncher-joplin#2]
Hi there,

The single [issue](https://github.com/KuenzelIT/ulauncher-joplin/issues/1) (opened in 2020) in this repo still prevails. Is the project alive? I'm missing detailed installation instructions in the `README`, in particular the exact `pip` commands to install the requirements.

Cheers!

**Comment by KuenzelIT** [comment]
Hey,

sorry, I switched to the albert launcher for different reasons, so I'm not using ulauncher anymore. I'm not sure if the extension still works. So the project is unfortunately not maintained anymore.

**Comment by gerardbosch** [comment]
Thanks for your reply! I tried albert some time ago and it didn't convince me. Maybe I can give it another try, but I'm very happy now with Ulauncher.

Just to be curious, can I ask the reasons that convinced you to switch? Thanks ^_^

**Comment by KuenzelIT** [comment]
I used the albert launcher before ulauncher, but I think the Firefox bookmarks plugin had some bugs that took a while to resolve. So I switched to ulauncher and liked the customization options there.

But then I realized the alber launcher had better performance and ulauncher didn't have such a good ssh integration. And as soon as the Firefox bookmarks plugin worked again, I switched back. :)

Would like to hear why albert didn't convince you? I really like how I can use ssh, firefox bookmarks and remote desktops (with remmina) in albert. Oh and snippets are pretty neat. Nearly everything I need.

**Comment by gerardbosch** [comment]
Hi there! I can't remember right now what moved me from Albert to Ulauncher :-S I was testing it maybe a couple of weeks and don't remember what missed or happened to decide to explore other options.

I'm using also Remmina with Ulauncher, raindrop.io (which I recently discovered) integration to open bookmarks with the default browser, and all sort of extensions:

![image](https://user-images.githubusercontent.com/30733556/198556659-a39b22ba-712e-407a-ab76-f09b4e0ba583.png)

Possibly Albert offers better performance or memory footprint? I'd like to remember why I shifted... :sweat_smile: Maybe I give it a new shot in the future.

Thanks to share your impressions!! :smiley:
